src/exabgp/reactor/api/command/neighbor.py

```diff
@@ -194,22 +183,23 @@ def teardown(self, reactor, service, line):
         return False
 
 
-@Command.register('text', 'show neighbor', False, ['summary', 'extensive', 'configuration', 'json'])
+@Command.register('text', 'show neighbor', False, ['summary', 'extensive', 'configuration'])
+@Command.register('text', 'show neighbor json', False, ['summary', 'extensive'])
```

Comment:
if we register <command> json, we should also register <command> text and make it work.